packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/UpdateWorkingGroupBudget.tsx

```diff
@@ -81,25 +102,42 @@ export const UpdateWorkingGroupBudget = () => {
           {group && (
             <Info>
               <TextMedium>
-                Current budget for {capitalizeFirstLetter(group.name)} Working Group is{' '}
+                Current budget for {nameMapping(group.name)} Working Group is{' '}
                 <TextInlineMedium bold>
                   <TokenValue value={group.budget} />
                 </TextInlineMedium>
                 .
               </TextMedium>
             </Info>
           )}
-          {councilBudget && (
+          {budget.amount && (
             <Info>
               <TextMedium>
                 Current budget for Council is{' '}
                 <TextInlineMedium bold>
-                  <TokenValue value={councilBudget} />
+                  <TokenValue value={budget.amount} />
                 </TextInlineMedium>
                 .
               </TextMedium>
             </Info>
           )}
+          {nextPaymentBlock && milisecondsLeft && (
```

Comment:
Just to make sure to not render "0":
```suggestion
          {nextPaymentBlock && !!milisecondsLeft && (
```